### ISSUE TITLE
Can not launch the target in new powershell terminal

### ISSUE THREAD

**Issue by IEXPECTATION** [microsoft/vscode-cmake-tools#2621]
### Brief Issue Summary

I can't  launch the target in new powershell. The instruction to be executed is wrapped in quotation marks, I think PowerShell treats it as a string, are there something wrong about my configuration?

The powershell version is  7.2.5.
The generator I use is "Msys Makefiles".

### CMake Tools Diagnostics

```shell
{
  "os": "win32",
  "vscodeVersion": "1.68.1",
  "cmtVersion": "1.12.4",
  "configurations": [
    {
      "folder": "c:\\msys64\\home\\11206\\dev\\project-xin\\xflag",
      "cmakeVersion": "3.23.2",
      "configured": false,
      "generator": "MinGW Makefiles",
      "usesPresets": false,
      "compilers": {}
    }
  ],
  "cpptoolsIntegration": {
    "isReady": false,
    "hasCodeModel": false,
    "activeBuildType": "",
    "buildTypesSeen": [],
    "requests": [],
    "responses": [],
    "partialMatches": [],
    "targetCount": 0,
    "executablesCount": 0,
    "librariesCount": 0,
    "targets": []
  },
  "settings": [
    {
      "communicationMode": "automatic",
      "useCMakePresets": "auto",
      "configureOnOpen": true
    }
  ]
}
```

### Debug Log

```shell
[extension] [7684] cmake.launchTarget started
[main] Building folder: xflag xflag
[main] Saving open files before configure/build
[build] Starting build
[driver] Start build xflag
[proc] Executing command: C:/msys64/clang64/bin/cmake.exe --build c:/msys64/home/11206/dev/project-xin/xflag/build --config Debug --target xflag -j 48 --
[build] [100%] Built target xflag
[cmakefileapi-parser] Read reply folder: c:\msys64\home\11206\dev\project-xin\xflag\build\.cmake\api\v1\reply
[cmakefileapi-parser] Found index files: ["cache-v2-644ab7cd653c72bef0e1.json","codemodel-v2-fe937cd02859a68f41cf.json","directory-.-Debug-d0094a50bb2071803777.json","index-2022-06-25T14-31-23-0331.json","target-xflag-Debug-9353abb2c919af2973ac.json","toolchains-v1-74873b4e6f6ae83b0148.json"]
[build] Build finished with exit code 0
[extension] [7684] cmake.launchTarget finished (returned {"name":"CMake/Launch","processId":{},"creationOptions":{"name":"CMake/Launch","env":{"!;":";\\","ACLOCAL_PATH":"C:\\msys64\\ucrt64\\share\\aclocal;C:\\msys64\\usr\\share\\aclocal","ALLUSERSPROFILE":"C:\\ProgramData","APPDATA":"C:\\Users\\11206\\AppData\\Roaming","APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL":"true","CABAL_DIR":"C:\\Haskell\\cabal","CHROME_CRASHPAD_PIPE_NAME":"\\\\.\\pipe\\crashpad_22012_WNZRWIHLVVORYKZJ","COMMONPROGRAMFILES":"C:\\Program Files\\Common Files","CommonProgramFiles(x86)":"C:\\Program Files (x86)\\Common Files","CommonProgramW6432":"C:\\Program Files\\Common Files","COMPUTERNAME":"LAPTOP-A5NQ8BQH","COMSPEC":"C:\\WINDOWS\\system32\\cmd.exe","CONFIG_SITE":"C:/msys64/etc/config.site","CONICON":"ucrt64.ico","CONTITLE":"MinGW UCRT x64","cp":"D:/Mycode/JJar","DriverData":"C:\\Windows\\System32\\Drivers\\DriverData","ELECTRON_NO_ATTACH_CONSOLE":"1","ELECTRON_RUN_AS_NODE":"1","FPS_BROWSER_APP_PROFILE_STRING":"Internet Explorer","FPS_BROWSER_USER_PROFILE_STRING":"Default","GHCUP_INSTALL_BASE_PREFIX":"C:\\Haskell\\","GHCUP_MSYS2":"C:\\msys64","GOBIN":"C:\\gotools\\bin","GOPATH":"C:\\Users\\11206\\go","GOROOT":"C:\\Program Files\\Go\\","GRADLE_USER_HOME":"C:\\users\\11206\\.gradle","HOME":"C:\\msys64\\home\\11206","HOMEDRIVE":"C:","HOMEPATH":"\\Users\\11206","HOSTNAME":"LAPTOP-A5NQ8BQH","INFOPATH":"C:\\msys64\\usr\\local\\info;C:\\msys64\\usr\\share\\info;C:\\msys64\\usr\\info;C:\\msys64\\share\\info","INTEL_DEV_REDIST":"C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\","JAVA_HOME":"D:\\JDK\\JDK11","JDK11":"D:\\JDK\\JDK11","JDK17":"D:\\JDK\\JDK17","LANG":"en_US.UTF-8","LOCALAPPDATA":"C:\\Users\\11206\\AppData\\Local","LOGINSHELL":"bash","LOGONSERVER":"\\\\LAPTOP-A5NQ8BQH","MANPATH":"C:\\msys64\\ucrt64\\local\\man;C:\\msys64\\ucrt64\\share\\man;C:\\msys64\\usr\\local\\man;C:\\msys64\\usr\\share\\man;C:\\msys64\\usr\\man;C:\\msys64\\share\\man","MIC_LD_LIBRARY_PATH":"C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\compiler\\lib\\mic","MINGW_CHOST":"x86_64-w64-mingw32","MINGW_PACKAGE_PREFIX":"mingw-w64-ucrt-x86_64","MINGW_PREFIX":"C:/msys64/ucrt64","Msys2_Home":"C:\\msys64\\home\\11206","MSYS2_NOSTART":"yes","MSYS2_PATH_TYPE":"inherit","MSYSTEM":"UCRT64","MSYSTEM_CARCH":"x86_64","MSYSTEM_CHOST":"x86_64-w64-mingw32","MSYSTEM_PREFIX":"C:/msys64/ucrt64","NUMBER_OF_PROCESSORS":"8","NVIDIAWHITELISTED":"0x01","OLDPWD":"C:/msys64/home/11206/dev/project-xin/xflag/build","OneDrive":"C:\\Users\\11206\\OneDrive","ORIGINAL_PATH":"C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\redist\\intel64\\compiler;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\iCLS;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\iCLS;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\WINDOWS\\System32\\OpenSSH;D:\\COMMAND;C:\\msys64\\usr\\bin;C:\\msys64\\ucrt64\\bin;C:\\msys64\\mingw64\\bin;C:\\msys64\\clang64\\bin;C:\\Program Files\\dotnet;C:\\Program Files\\Microsoft SQL Server\\130\\Tools\\Binn;D:\\maven\\bin;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\Program Files\\Microsoft SQL Server\\140\\Tools\\Binn;C:\\Program Files\\Microsoft SQL Server\\150\\Tools\\Binn;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\170\\Tools\\Binn;C:\\Julia\\bin;C:\\Program Files\\MariaDB 10.7\\bin;C:\\gotools\\bin;D:\\Git\\cmd;C:\\Program Files\\nodejs;C:\\Program Files (x86)\\dotnet;C:\\Program Files\\Go\\bin;C:\\Users\\11206\\AppData\\Local\\Programs\\Python\\Python310\\Scripts;C:\\Users\\11206\\AppData\\Local\\Programs\\Python\\Python310;C:\\Users\\11206\\AppData\\Local\\Microsoft\\WindowsApps;D:\\Microsoft VS Code\\bin;C:\\Users\\11206\\.dotnet\\tools;C:\\Users\\11206\\AppData\\Local\\Microsoft\\WindowsApps;D:\\MPVPlayer;C:\\Users\\11206\\.dotnet\\tools;C:\\tools\\ghc-9.2.1\\bin;C:\\Haskell\\ghcup\\bin;C:\\Users\\11206\\.dotnet\\tools;C:\\Users\\11206\\AppData\\Roaming\\npm;C:\\Users\\11206\\go\\bin","ORIGINAL_TEMP":"C:/Users/11206/AppData/Local/Temp","ORIGINAL_TMP":"C:/Users/11206/AppData/Local/Temp","ORIGINAL_XDG_CURRENT_DESKTOP":"undefined","OS":"Windows_NT","PATH":"C:\\msys64\\ucrt64\\bin;C:\\msys64\\usr\\local\\bin;C:\\msys64\\usr\\bin;C:\\msys64\\usr\\bin;C:\\Program Files (x86)\\Common Files\\Intel\\Shared Libraries\\redist\\intel64\\compiler;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\iCLS;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\iCLS;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\WINDOWS\\System32\\OpenSSH;D:\\COMMAND;C:\\msys64\\usr\\bin;C:\\msys64\\ucrt64\\bin;C:\\msys64\\mingw64\\bin;C:\\msys64\\clang64\\bin;C:\\Program Files\\dotnet;C:\\Program Files\\Microsoft SQL Server\\130\\Tools\\Binn;D:\\maven\\bin;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\Program Files\\Microsoft SQL Server\\140\\Tools\\Binn;C:\\Program Files\\Microsoft SQL Server\\150\\Tools\\Binn;C:\\Program Files\\Microsoft SQL Server\\Client SDK\\ODBC\\170\\Tools\\Binn;C:\\Julia\\bin;C:\\Program Files\\MariaDB 10.7\\bin;C:\\gotools\\bin;D:\\Git\\cmd;C:\\Program Files\\nodejs;C:\\Program Files (x86)\\dotnet;C:\\Program Files\\Go\\bin;C:\\Users\\11206\\AppData\\Local\\Programs\\Python\\Python310\\Scripts;C:\\Users\\11206\\AppData\\Local\\Programs\\Python\\Python310;C:\\Users\\11206\\AppData\\Local\\Microsoft\\WindowsApps;D:\\Microsoft VS Code\\bin;C:\\Users\\11206\\.dotnet\\tools;C:\\Users\\11206\\AppData\\Local\\Microsoft\\WindowsApps;D:\\MPVPlayer;C:\\Users\\11206\\.dotnet\\tools;C:\\tools\\ghc-9.2.1\\bin;C:\\Haskell\\ghcup\\bin;C:\\Users\\11206\\.dotnet\\tools;C:\\Users\\11206\\AppData\\Roaming\\npm;C:\\Users\\11206\\go\\bin;C:\\msys64\\usr\\bin\\site_perl;C:\\msys64\\usr\\bin\\vendor_perl;C:\\msys64\\usr\\bin\\core_perl;C:\\msys64\\clang64\\bin","PATHEXT":".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC","PKG_CONFIG_PATH":"C:\\msys64\\ucrt64\\lib\\pkgconfig;C:\\msys64\\ucrt64\\share\\pkgconfig","PRINTER":"Microsoft Print to PDF","PROCESSOR_ARCHITECTURE":"AMD64","PROCESSOR_IDENTIFIER":"Intel64 Family 6 Model 158 Stepping 10, GenuineIntel","PROCESSOR_LEVEL":"6","PROCESSOR_REVISION":"9e0a","ProgramData":"C:\\ProgramData","PROGRAMFILES":"C:\\Program Files","ProgramFiles(x86)":"C:\\Program Files (x86)","ProgramW6432":"C:\\Program Files","PROMPT":"$P$G","PSModulePath":"C:\\Program Files\\WindowsPowerShell\\Modules;C:\\WINDOWS\\system32\\WindowsPowerShell\\v1.0\\Modules","PUBLIC":"C:\\Users\\Public","PWD":"C:/msys64/home/11206/dev/project-xin/xflag","SESSIONNAME":"Console","SHELL":"C:\\msys64\\usr\\bin\\bash.exe","SHIM_MCCOMPAT":"0x810000001","SHLVL":"2","SYSTEMDRIVE":"C:","SYSTEMROOT":"C:\\WINDOWS","TEMP":"C:\\msys64\\tmp","TERM":"xterm-256color","TMP":"C:\\msys64\\tmp","USER":"11206","USERDOMAIN":"LAPTOP-A5NQ8BQH","USERDOMAIN_ROAMINGPROFILE":"LAPTOP-A5NQ8BQH","USERNAME":"11206","USERPROFILE":"C:\\Users\\11206","VSCODE_AMD_ENTRYPOINT":"vs/workbench/api/node/extensionHostProcess","VSCODE_CLI":"1","VSCODE_CODE_CACHE_PATH":"C:\\Users\\11206\\AppData\\Roaming\\Code\\CachedData\\30d9c6cd9483b2cc586687151bcbcd635f373630","VSCODE_CWD":"C:\\msys64\\home\\11206\\dev\\project-xin\\xflag","VSCODE_HANDLES_UNCAUGHT_ERRORS":"true","VSCODE_IPC_HOOK":"\\\\.\\pipe\\5166f80ebf50375b3f45b3bb2b2afb28-1.68.1-main-sock","VSCODE_IPC_HOOK_EXTHOST":"\\\\.\\pipe\\vscode-ipc-3370af07-ce8d-4486-be5a-a6b28cad798e-sock","VSCODE_LOG_NATIVE":"false","VSCODE_LOG_STACK":"false","VSCODE_NLS_CONFIG":"{\"locale\":\"en\",\"availableLanguages\":{},\"_languagePackSupport\":true}","VSCODE_PID":"22012","VSCODE_PIPE_LOGGING":"true","VSCODE_VERBOSE_LOGGING":"true","VSP":"C:\\Users\\11206\\source\\repos","WD":"C:\\msys64\\usr\\bin\\","WINDIR":"C:\\WINDOWS","WSLENV":"WT_SESSION::WT_PROFILE_ID","WT_PROFILE_ID":"{75b66e97-bd40-4ff3-95b1-d02f2d6cd869}","WT_SESSION":"82c02df8-7288-4d09-a240-695ad97eb779","XDG_DATA_DIRS":"C:\\msys64\\ucrt64\\share\\;C:\\msys64\\usr\\local\\share\\;C:\\msys64\\usr\\share\\","ZES_ENABLE_SYSMAN":"1","_":"D:/Microsoft VS Code/Code.exe","_CMAKE_TOOLS_LAUNCH_TERMINAL_TARGET_NAME":"xflag","_CMAKE_TOOLS_LAUNCH_TERMINAL_PATH":"C:\\Users\\11206\\.dotnet\\tools\\pwsh.exe"},"cwd":"${workspaceFolder}"},"state":{"isInteractedWith":true},"dimensions":{"columns":114,"rows":14}})
```

### Additional Information

![Screenshot 2022-06-25 223756](https://user-images.githubusercontent.com/40608137/175778602-0abf5ef9-d9cf-42cb-9b6d-819f96e7b130.png)

**Comment by bobbrow** [comment]
This looks like a duplicate of #535 which appears to have been around for a long time. I thought this had been addressed with #2311, but maybe more works needs to be done for it.  We'll track any remaining Powershell issues with #535 though.

**Comment by LucasOe** [comment]
Duplicate of #535. There's a dot missing before the path because of which the path gets printed instead of executed.
